From a7018654ad6d454e11b8e6361de3c83f439a2466 Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Thu, 10 Aug 2017 16:16:20 -0400
Subject: [PATCH 1814/4131] drm/amd/display: Leave DCN config guard around fpu
 header

See
https://lists.freedesktop.org/archives/dri-devel/2017-August/149938.html

Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Reviewed-by: Jordan Lazare <Jordan.Lazare@amd.com>
Acked-by: Harry Wentland <Harry.Wentland@amd.com>

 Conflicts:
        drivers/gpu/drm/amd/display/dc/dm_services.h

Change-Id: I9ed7aa808d4f28ead846e686ea3f18f03b2975b1
---
 drivers/gpu/drm/amd/display/dc/dm_services.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/drivers/gpu/drm/amd/display/dc/dm_services.h b/drivers/gpu/drm/amd/display/dc/dm_services.h
index a7d661d..1782582 100644
--- a/drivers/gpu/drm/amd/display/dc/dm_services.h
+++ b/drivers/gpu/drm/amd/display/dc/dm_services.h
@@ -75,7 +75,12 @@
 	BREAK_TO_DEBUGGER(); \
 } while (0)
 
+#if defined(CONFIG_DRM_AMD_DC_DCN1_0)
 #include <asm/fpu/api.h>
+#endif
+#else
+#include <asm/i387.h>
+#endif
 
 #define dm_alloc(size) kzalloc(size, GFP_KERNEL)
 #define dm_realloc(ptr, size) krealloc(ptr, size, GFP_KERNEL)
-- 
2.7.4

